packages/nextjs-demo/src/app/basic-completion/api/text-stream/route.tsx

```diff
@@ -0,0 +1,25 @@
+/** @jsxImportSource ai-jsx */
+import { toTextStream } from 'ai-jsx/stream';
+import { NextRequest } from 'next/server';
+import { ChatCompletion, UserMessage } from 'ai-jsx/core/completion';
+import { StreamingTextResponse } from 'ai';
+
+export async function POST(request: NextRequest) {
+  const { topic } = await request.json();
+
+  return new StreamingTextResponse(
```

Comment:
Is this actually doing anything for us?

Reply:
Not really, I just wanted to use it to show that it works with their AI SDK. Originally I was just creating a response myself but when I saw that their RSC streamer takes a `ReadableStream` I opted to make that the abstraction instead.